### DISCUSSION TITLE
Installed addon-docs but no Docs tab showing

### DISCUSSION THREAD

**Discussion by JESii** [storybookjs/storybook#14989] [Documentation]
I installed the docs add-on as described here:[https://storybook.js.org/addons/@dblechoc/storybook-addon-docs]( https://storybook.js.org/addons/@dblechoc/storybook-addon-docs). It says that `When you install Docs, every story gets a DocsPage`, but that didn't happen. I don't see the Docs tab that should have appeared.

We're running 
     "@storybook/addon-docs": "^6.2.9",
    "@storybook/addon-essentials": "^6.2.1",
    "@storybook/react": "^6.2.1",
Any suggestions? Thanks.

**Comment by shilman** [comment]
Can you please create a reproduction by running `npx sb@next repro`, following the instructions, and linking it in your issue description? We prioritize issues with reproductions over those without. Thank you! 🙏
